decrypt_oracle/tox.ini

```
@@ -35,7 +35,7 @@ envlist =
 
 
 [testenv:generate-pipeline]
-basepython = python3.9
+basepython = python3
```

Revert: Python3.9 is the latest version of Python AWS Lambda supports.
It behooves us to use the latest Python available, 
so we do not have to re-deploy the Lambda again when Python3.7 or 3.8 goes EOL.
```suggestion
basepython = python3.9
```